_pages/en/employee-profiles.md

```diff
@@ -21,7 +21,7 @@ These profiles are to help categorize employees and determine their IT and busin
 - Client services - Work directly with the public
 - Call Center User - Similar to an activity-based worker but requires additional telephony options
 - General GC Office users - Usually at their desk
-- Special needs/Accessible - worker with disabilities in the workplace who requires a client’s need assessment and awareness services   for adaptive technology
+- Special needs/Accessible - worker with disabilities in the workplace who requires a client's need assessment and awareness services   for adaptive technology
```

Comment:
I fully approve changing all ` to ' :confetti_ball:

Reply:
Even though I believe ` is valid in English, it wasn't recognized by the spellchecker, thus generating false positives.